src/entity/ChatMessage.java

```diff
@@ -0,0 +1,9 @@
+package entity;
```

Comment:
Oops, we now have a Mesage.java entity, sorry for leaving this hanging for so long!

Reply:
Ok, resolved by making chatmessage child of message.